test/Common/MoistThermodynamics/runtests.jl

```diff
@@ -504,7 +504,7 @@ end
         @test all(isapprox.(
             internal_energy.(ts),
             internal_energy.(ts_exact),
-            rtol = rtol,
+            rtol = rtol*2,
```

Comment:
I did have to increase the tolerance for this test (for the `LiquidIcePotTempSHumNonEquil` constructor)